### PR TITLE
Remove using namespace std; from appleseed.studio

### LINKED ISSUE #2641: Remove using namespace std directives

### PR DESCRIPTION
**Pull request by LZaw**
This PR part of a series.
Fixes #2641 for appleseed.studio.

See also: #2689, #2691, #2692, #2693, #2694

### CODE REVIEW

File: src/appleseed.studio/mainwindow/project/disneymaterialcustomui.cpp [hunk 1]
```diff
@@ -76,14 +75,14 @@ namespace
         }
     };
 
-    vector<Dictionary> collect_layer_definitions(const Dictionary& values)
+    std::std::vector<Dictionary> collect_layer_definitions(const Dictionary& values)
```
Comment (dictoon) [inline src/appleseed.studio/mainwindow/project/disneymaterialcustomui.cpp]: Double qualifier.

File: src/appleseed.studio/mainwindow/pythonconsole/pythonconsolewidget.cpp [hunk 2]
```diff
@@ -321,32 +320,32 @@ void PythonConsoleWidget::open_file(const string& filepath)
 
     while (!file.eof())
     {
-        string str;
+        std::string str;
         getline(file, str);
```
Comment (dictoon) [inline src/appleseed.studio/mainwindow/pythonconsole/pythonconsolewidget.cpp]: That should be qualified as well I think.

File: src/appleseed.studio/mainwindow/rendering/lightpathswidget.cpp [hunk 3]
```diff
@@ -90,7 +89,7 @@ namespace {
     struct OpenGLRasterizer
       : public ObjectRasterizer
     {
-        vector<float>   m_buffer;
+        std::vector<float>   m_buffer;
```
Comment (dictoon) [inline src/appleseed.studio/mainwindow/rendering/lightpathswidget.cpp]: Fix alignment.

File: src/appleseed.studio/mainwindow/renderingsettingswindow.cpp [hunk 4]
```diff
@@ -396,7 +395,7 @@ class RenderSettingsPanel
 
     template <typename T>
     void set_widget(
-        const string&           widget_key,
+        const std::string&           widget_key,
```
Comment (dictoon) [inline src/appleseed.studio/mainwindow/renderingsettingswindow.cpp]: Fix indentation.

File: src/appleseed.studio/mainwindow/renderingsettingswindow.cpp [hunk 5]
```diff
@@ -406,7 +405,7 @@ class RenderSettingsPanel
     template <typename T>
     static T get_config(
         const Configuration&    configuration,
-        const string&           param_path,
+        const std::string&           param_path,
```
Comment (dictoon) [inline src/appleseed.studio/mainwindow/renderingsettingswindow.cpp]: Fix indentation.

File: src/appleseed.studio/mainwindow/renderingsettingswindow.cpp [hunk 6]
```diff
@@ -416,7 +415,7 @@ class RenderSettingsPanel
     template <typename T>
     static void set_config(
         Configuration&          configuration,
-        const string&           param_path,
+        const std::string&           param_path,
```
Comment (dictoon) [inline src/appleseed.studio/mainwindow/renderingsettingswindow.cpp]: Fix indentation.

File: src/appleseed.studio/utility/widgetzoomhandler.cpp [hunk 7]
```diff
@@ -173,18 +172,18 @@ void WidgetZoomHandler::reset_zoom()
 void WidgetZoomHandler::compute_min_max_scale_factors()
 {
     m_min_scale_factor =
-        max(
-            MinScaleFactor,
-            max(
-                MinWidgetSize / m_content_width,
-                MinWidgetSize / m_content_height));
+        std::max(
+                 MinScaleFactor,
```
Comment (dictoon) [inline src/appleseed.studio/utility/widgetzoomhandler.cpp]: Fix indentation (4 spaces).

File: src/appleseed.studio/mainwindow/project/disneymaterialcustomui.cpp (hunk 1 above)
Comment (LZaw) [reply src/appleseed.studio/mainwindow/project/disneymaterialcustomui.cpp]: Oh, sorry. Interesting it compiles anyways...

File: src/appleseed.studio/mainwindow/pythonconsole/pythonconsolewidget.cpp (hunk 2 above)
Comment (LZaw) [reply src/appleseed.studio/mainwindow/pythonconsole/pythonconsolewidget.cpp]: You're right. I will also look out for other `getline`s too.

**Pull request merged**